### PR TITLE
Review submission fix

### PR DESCRIPTION
**Pull request by nully0x**
This PR fixes:
- pr_url not saving in db thereby causing submission to fail.
- optimise the review submission logic

### CODE REVIEW

File: api/app/db/models/review.ts [hunk 1]
```diff
@@ -34,6 +34,9 @@ export class Review extends Model<ReviewAttributes> {
   @Column(DataType.DATE)
   public mergedAt?: Date;
 
+  @Column(DataType.STRING)
+  public pr_url!: string;
```
Comment (Extheoisah) [inline api/app/db/models/review.ts:38]: pr_url is not a mandatory field. It can be null

File: api/app/controllers/review.controller.ts [hunk 2]
```diff
@@ -190,32 +190,35 @@ export async function submit(req: Request, res: Response) {
   const id = req.params.id;
   const { pr_url } = req.body;
 
+  console.log("pr_url", pr_url);
+  console.log("id", id);
```
Comment (Emmanuel-Develops) [inline api/app/controllers/review.controller.ts:194]: Remove logs

Comment (nully0x) [reply api/app/controllers/review.controller.ts:194]: > Remove logs

gotcha

**Review by Extheoisah — APPROVED** [review]

**Review by Extheoisah — APPROVED** [review]

**Comment by Extheoisah** [comment]
You added a merge commit. That won't fox the rebase conflict

**Pull request closed without merging**